### DISCUSSION TITLE
希望人工审核当设置有多个审核人时可以设置所有的同学都通过以后才向下执行

### DISCUSSION THREAD

**Discussion by Justice-love** [Tencent/bk-ci#6625] [Ideas]
需求：希望人工审核当设置有多个审核人时可以设置所有的同学都通过以后才向下执行

应用场景：流水线中需要多个同学通过审核后才好执行下一步、但是目前是其中一个同学通过后就继续向下执行了、所以希望增加这个选项

**Comment by zanyzhao** [comment]
Stage准入里已经增加了审批流，可以满足你的场景
